packages/@webex/plugin-meetings/src/locus-info/selfUtils.ts

```diff
@@ -72,27 +74,32 @@ SelfUtils.parse = (self: any, deviceId: string) => {
   return null;
 };
 
-SelfUtils.getBreakoutSessions = (self) => self?.controls?.breakout?.sessions;
-SelfUtils.getBreakout = (self) => self?.controls?.breakout;
-SelfUtils.getInterpretation = (self) => self?.controls?.interpretation;
+SelfUtils.getBreakoutSessions = (self: Record<string, any>) => self?.controls?.breakout?.sessions;
```

Comment:
Btw, I already have a request that I will add to review later for updating with the proper types of locus utils.